FAIL: TestContentClient/Resume (0.03s)

The build resulting from me merging https://github.com/containerd/containerd/pull/1822 failed with the following:

```
 --- FAIL: TestContentClient/Resume (0.03s)
    	testsuite.go:183: compare update 2017-11-28 23:00:11.101380553 +0000 UTC against (2017-11-28 23:00:11.100776457 +0000 UTC m=+8.351542785, 2017-11-28 23:00:11.10193697 +0000 UTC m=+8.352703301)
    	testsuite.go:192: compare update 2017-11-28 23:00:11.101380553 +0000 UTC against (2017-11-28 23:00:11.100776457 +0000 UTC m=+8.351542785, 2017-11-28 23:00:11.10193697 +0000 UTC m=+8.352703301)
    	testsuite.go:200: compare update 2017-11-28 23:00:11.110823652 +0000 UTC against (2017-11-28 23:00:11.110620775 +0000 UTC m=+8.361387109, 2017-11-28 23:00:11.111068218 +0000 UTC m=+8.361834561)
    	helpers.go:52: drwx------       4096 /tmp/content-suite-ContentClient-237962037
    	testsuite.go:46: Cleanup failed: not found
    		github.com/containerd/containerd/errdefs.init
    			/home/travis/gopath/src/github.com/containerd/containerd/errdefs/errors.go:25
    		github.com/containerd/containerd/content.init
    			<autogenerated>:1
    		github.com/containerd/containerd.init
    			<autogenerated>:1
    		main.init
    			<autogenerated>:1
    		runtime.main
    			/home/travis/.gimme/versions/go1.9.2.linux.amd64/src/runtime/proc.go:183
    		runtime.goexit
    			/home/travis/.gimme/versions/go1.9.2.linux.amd64/src/runtime/asm_amd64.s:2337
    		content digest sha256:daec4241907ca188cedc84c1efc664ef5a0989820fe22de189c6e219603932d5
    		github.com/containerd/containerd/errdefs.FromGRPC
    			/home/travis/gopath/src/github.com/containerd/containerd/errdefs/grpc.go:82
    		github.com/containerd/containerd.(*remoteContent).Delete
    			/home/travis/gopath/src/github.com/containerd/containerd/content_store.go:68
    		github.com/containerd/containerd.newContentStore.func1.1
    			/home/travis/gopath/src/github.com/containerd/containerd/content_test.go:31
    		github.com/containerd/containerd.(*remoteContent).Walk
    			/home/travis/gopath/src/github.com/containerd/containerd/content_store.go:55
    		github.com/containerd/containerd.newContentStore.func1
    			/home/travis/gopath/src/github.com/containerd/containerd/content_test.go:30
    		github.com/containerd/containerd/content/testsuite.makeTest.func1.1
    			/home/travis/gopath/src/github.com/containerd/containerd/content/testsuite/testsuite.go:45
    		github.com/containerd/containerd/content/testsuite.makeTest.func1
    			/home/travis/gopath/src/github.com/containerd/containerd/content/testsuite/testsuite.go:52
    		testing.tRunner
    			/home/travis/.gimme/versions/go1.9.2.linux.amd64/src/testing/testing.go:746
    		runtime.goexit
    			/home/travis/.gimme/versions/go1.9.2.linux.amd64/src/runtime/asm_amd64.s:2337
```
Travis: https://travis-ci.org/containerd/containerd/jobs/308695150

/cc @dmcgowan

--- a/content_test.go
+++ b/content_test.go
@@ -6,6 +6,7 @@ import (
 
 	"github.com/containerd/containerd/content"
 	"github.com/containerd/containerd/content/testsuite"
+	"github.com/containerd/containerd/errdefs"
 	"github.com/pkg/errors"
 )
 
@@ -28,7 +29,14 @@ func newContentStore(ctx context.Context, root string) (content.Store, func() er
 			}
 		}
 		return cs.Walk(ctx, func(info content.Info) error {
-			return cs.Delete(ctx, info.Digest)
+			if err := cs.Delete(ctx, info.Digest); err != nil {
+				if errdefs.IsNotFound(err) {
+					return nil
+				}
+
+				return err
+			}
+			return nil
 		})
 
 	}, nil

